File: 0026_Remove_Duplicates_From_Sorted_Array/0026_Remove_Duplicates_From_Sorted_Array.ts
function removeDuplicates(nums: number[]): number {
    var i = 0;

    for (let j = 1; j<nums.length; j++) {
        // Element not equal, then save it to the right position
        if (nums[i] !== nums[j]) {
            i = i + 1;
            nums[i] = nums[j];
        }
    }

    return i + 1;
}

File: 0026_Remove_Duplicates_From_Sorted_Array/0026_Remove_Duplicates_From_Sorted_Array.swift
class Solution {
    func removeDuplicates(_ nums: inout [Int]) -> Int {
        var i = 0;

        for j in 1..<nums.count {
            if nums[i] != nums[j] {
                i = i + 1
                nums[i] = nums[j]
            }
        }

        return i + 1
    }
}